erts/doc/src/notes.xml

```diff
@@ -16200,7 +16200,7 @@
           <p>
 	    The effect of the deprecated environment variable
 	    ERL_MAX_PORTS had been removed premeturely. It has now
-	    been readded. Note that this is still scheduled to be
+	    been read. Note that this is still scheduled to be
```

Comment:
```suggestion
	    been re-added. Note that this is still scheduled to be
```

Reply:
While at it, fix misspelled "prem**a**turely" in the line above.